doc/development/deprecations.rst

```diff
@@ -6,6 +6,18 @@ Deprecations
 Pending deprecations
 --------------------
 
+* ``qml.enable_return`` and ``qml.disable_return`` are deprecated. Please ensure that you are using the new
+  return system, as the old return system is deprecated along with these switch functions.
```

Comment:
> Please ensure you are using the new return system

If I read this I would think I needed to do something, but I wouldn't be sure what. Could clarify to add "The new return system is enabled by default in v0.32" so I know that the only thing I need to do is make sure my version of PennyLane is up to date.